test/ejabberd_tests/tests/last_SUITE.erl

```diff
@@ -61,9 +61,10 @@ end_per_testcase(CaseName, Config) ->
 %%--------------------------------------------------------------------
 last_online_user(Config) ->
     escalus:story(Config, [{alice, 1}, {bob, 1}],
-                  fun(Alice, _Bob) ->
+                  fun(Alice, Bob) ->
                           %% Alice asks about Bob's last activity
-                          escalus_client:send(Alice, escalus_stanza:last_activity(bob)),
+                          BobShortJID = escalus_client:short_jid(Bob),
```

Comment:
First of all it adds visible correctness. What we need in this function is `short_jid` in fact. The previous version returned short jid but only because we read the user spec from `ct:config`.
There are more details about deprecating the previous version here: https://github.com/esl/escalus/commit/93ee6eef1cce80ae7d1213c62b31cfebe301b11f#diff-2a28f470b6906a7c45b97b29f4e6a96eR138